tardis/plasma/properties/nlte.py

```diff
@@ -117,7 +117,7 @@ def calculate_helium_one(
         """
         return (
             level_boltzmann_factor.loc[2, 0]
-            * (1.0 / (2 * g.loc[2, 1, 0]))
```

Comment:
is this dangerous to assume it is 2

Reply:
I would hope that all atomic datasets would be consistent for such a low energy level, but I agree it's not very elegant. I will experiment and try to figure out why the issue is occurring in the first place.